conf/igenomes.config

```diff
@@ -96,6 +105,7 @@
       bowtie  = "${params.igenomes_base}/Glycine_max/Ensembl/Gm01/Sequence/BowtieIndex/genome"
       fasta   = "${params.igenomes_base}/Glycine_max/Ensembl/Gm01/Sequence/WholeGenomeFasta"
       gtf     = "${params.igenomes_base}/Glycine_max/Ensembl/Gm01/Annotation/Genes/genes.gtf"
+      refgenome   = "${params.igenomes_base}/Glycine_max/Ensembl/Gm01/Sequence/WholeGenomeFasta/genome.fa"
```

Comment:
Sorry, I'm not familiar with igenomes, so maybe this is from ignorance - what is the difference between the `fasta` and `refgenome` entries in this section? They look very similar?

Reply:
I will change the `--refgenome` and maintain the `fasta`, adding `/genome.fa` at the end, to retreive only the genome in fasta format